### DISCUSSION TITLE
[firebase-ml-natural-language] language-id-model and language-translate-model

### DISCUSSION THREAD

**Discussion by XinyueZ** [firebase/flutterfire#5572] [Feature request]
language-id-model
language-translate-model

#269 mybe?

**Comment by iapicca** [comment]
Hi @XinyueZ 
could you please describe your proposal more in details?
thank you

**Comment by XinyueZ** [comment]
@iapicca 
The Flutter plugin for firebase-ml-natural-language.
firebase_ml_vision: This is what currently is built:
BarcodeModel
FaceModel
LabelModel
TextModel

But there is still natural language parts in ML-kit:
lang-id
on device tranlsation
i.e: https://firebase.google.com/docs/ml-kit/translation https://firebase.google.com/docs/ml-kit/identify-languages

**Comment by iapicca** [comment]
Hi @XinyueZ 
thank you for the clarification
